res/database/Default/database_programmes.xml

```diff
@@ -9666,60 +9666,59 @@ Dream scenes and sentiments warp the storyline.</en>
 				<member index="3" function="2">TheRob-TowerTV-PhilliGemoy</member>
 			</staff>
 			<groups target_groups="0" pro_pressure_groups="0" contra_pressure_groups="0" />
-			<data country="I" year="1976" distribution="2" maingenre="2" subgenre="" flags="0" blocks="1" price_mod="1.00" />
+			<data country="I" year="1976" distribution="2" maingenre="2" subgenre="1,7" flags="0" blocks="1" price_mod="1.00" />
 			<ratings critics="42" speed="46" outcome="" />
 			<children>
-				<!-- No official original German episode titles found, applying translation from original English -->
 				<programme id="TheRob-serie-Sanddornkahn-Episode001" product="2" licence_type="2" tmdb_id="0" imdb_id="tt1306736" creator="8751" created_by="TheRob">
 					<title>
-						<de>Episode 1.1</de>
-						<en>Episode 1.1</en>
+						<de>Die Entführung</de>
+						<en>The Conquest</en>
 					</title>
 					<title_original>
-						<de>Die Entführung</de>
+						<de>Die Eroberer</de>
 						<en>The Kidnapping</en>
 					</title_original>
 					<description>
-						<de>S. erkennt die Ungerechtigkeiten der Briten und entscheidet sich für den Kampf.</de>
-						<en>S. recognizes the injustices of the British and decides to fight.</en>
+						<de>S. erkennt die Ungerechtigkeiten der Briten und entscheidet sich für den Kampf. Eine chinesische Dschunke unter Kontrolle der Engländer wird versenkt.</de>
+						<en>S. recognizes the injustices of the British and decides to fight. A Chinese junk under British control is sunk.</en>
 					</description>
 				</programme>
 					<programme id="TheRob-serie-Sanddornkahn-Episode002" product="2" licence_type="2" tmdb_id="0" imdb_id="tt1306737" creator="8751" created_by="TheRob">
 					<title>
-						<de>Episode 1.2</de>
-						<en>Episode 1.2</en>
+						<de>Der Geheimnisvolle Prinz</de>
```

Comment:
adjective lower case?

Reply:
[Title casing](https://en.wikipedia.org/wiki/Title_case) is handled very inconsistently in general in the game data, in both German in English. (But it's not much better in the real world either.)

Agree in this case though, the other episodes are also not title-cased in German.